ankaios_sdk/_protos/__init__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2024 Elektrobit Automotive GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This module contains the ankaios_sdk protobuf components.
+It contains the proto files and the generated protobuf classes.
```

Comment:
I'll see what I can do. I believe the protos are published as individual assets of the ankaios release, so I should theoretically be able to get them fairly easy.

Reply:
Done. I added a method to extract the proto files directly from the release branch, requiring only the version of the release. Currently, it waits for the 0.5 release (because this is when the sdk will come out as well) and the changes needed to enable the automated fetching, after that point, are minimal.